x/sisu/client/rest/rest.go

```diff
@@ -1,15 +1,26 @@
 package rest
 
 import (
-	"github.com/gorilla/mux"
+	"net/http"
 
 	"github.com/cosmos/cosmos-sdk/client"
-	// this line is used by starport scaffolding # 1
+	"github.com/gorilla/mux"
+	"github.com/sisu-network/sisu/x/sisu"
 )
 
-const (
-	MethodGet = "GET"
-)
+type ApplicationHandler struct {
+	worldState sisu.WorldState
+}
+
+func NewApplicationHandler(worldState sisu.WorldState) *ApplicationHandler {
+	return &ApplicationHandler{
+		worldState: worldState,
+	}
+}
+
+func (a *ApplicationHandler) RegisterRoutes(clientCtx client.Context, r *mux.Router) {
+	r.HandleFunc("/getGasFeeInToken", a.newGasCostHandler()).Methods(http.MethodGet)
```

Comment:
Should we expose a API to get gas fee of all tokens? FE will get all when load page?